doc/dev/custom_devices.rst

```diff
@@ -181,3 +186,145 @@ After doing so, Catalyst should be able to interface with your custom device.
     @qml.qnode(CustomDevice(wires=1))
     def f():
         return measure(0)
+
+
+Below is an example configuration file
+
+.. code-block:: toml
+
+        schema = 1
+
+        [device]
+        name = "dummy.device.qubit"
```

Comment:
Does this need to correspond to anything (PL device short name for example, or the name used to find the device library, anything like that), or is it arbitrary?

Reply:
Also, since the configuration file is part of the device (it is the `config` field in `qml.Device`), the name of the file matching the device is also unnecessary.

Reply:
True, I guess we can leave as is then.